app/src/main/java/de/westnordost/streetcomplete/quests/air_pump/AddAirCompressor.kt

```diff
@@ -0,0 +1,49 @@
+package de.westnordost.streetcomplete.quests.air_pump
+
+import de.westnordost.streetcomplete.R
+import de.westnordost.streetcomplete.data.osm.mapdata.Element
+import de.westnordost.streetcomplete.data.osm.mapdata.MapDataWithGeometry
+import de.westnordost.streetcomplete.data.osm.mapdata.filter
+import de.westnordost.streetcomplete.data.osm.osmquests.OsmFilterQuestType
+import de.westnordost.streetcomplete.data.osm.osmquests.Tags
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement.BICYCLIST
+import de.westnordost.streetcomplete.data.user.achievements.QuestTypeAchievement.CAR
+import de.westnordost.streetcomplete.quests.YesNoQuestAnswerFragment
+import de.westnordost.streetcomplete.util.ktx.toYesNo
+import de.westnordost.streetcomplete.osm.updateWithCheckDate
+
+class AddAirCompressor : OsmFilterQuestType<Boolean>() {
+
+    override val elementFilter = """
+       nodes, ways with
+       amenity = fuel
+       and (
+           !compressed_air and !service:bicycle:pump
+           or compressed_air older today -6 years
+       )
+       and access !~ private|no
+    """
+    /* if service:bicycle:pump is undefined, nothing has been said about its existence;
+     * see https://wiki.openstreetmap.org/wiki/Tag:shop=bicycle#Additional_keys */
```

Comment:
I do not understand this comment. Why is this quest not asked if `service:bicycle:pump` is set to anything (no / yes)?

Reply:
sorry, ended in wrong file; it was meant to explain that undefined `service:bicycle:pump` does **not** imply that there is none (cf. https://github.com/streetcomplete/StreetComplete/pull/3934#pullrequestreview-927120024). Fixed it in e747679d7331e5b3e463fe955d2f2091deadca0d now.